docs/ossm/README.md

```diff
@@ -6,4 +6,54 @@ This documentation is specific to the OpenShift Service Mesh product and may dif
 
 # Table of Contents
 
-TODO
+- [Running OSSM 2 and OSSM 3 side by side](#running-ossm-2-and-ossm-3-side-by-side)
+
+
+# Running OSSM 2 and OSSM 3 side by side
+This section describes how to run OSSM 2 and OSSM 3 side by side in one cluster without interfering with each other.
+It's not OSSM 2 -> OSSM 3 migration guide.
```

Comment:
Maybe change this to:
**Note:** It's not OSSM 2 -> OSSM 3 migration guide.

Reply:
We can add the link later. Not sure if it make sense to add broken link now